src/main/java/qupath/ext/align/gui/ImageServerOverlay.java

```diff
@@ -60,6 +61,9 @@ public class ImageServerOverlay extends AbstractOverlay {
 	
 	private AffineTransform transform;
 	private AffineTransform transformInverse;
+
+	private PixelCalibration viewerCalibration;
```

Comment:
I'm not certain about the naming here - `viewerCalibrattion` sounds to me that it relates to the magnification in the viewer, not the image currently open within the viewer.

Not sure what the names should be, but maybe `currentServerCalibration` and `overlayServerCalibration`?

Or perhaps `viewerCalibration` and `overlayCalibration` would be fine... just not `serverCalibration` since it's not clear *which* server.

Reply:
I went with `viewerImageCalibration` and `overlayImageCalibration`, is that OK?